src/viewer/scene/scene/Scene.js

```diff
@@ -2562,6 +2567,63 @@ class Scene extends Component {
         return changed;
     }
 
+    /**
+     * This method will "tickify" the provided `cb` function.
+     * 
+     * This means, the function will be wrapped so:
+     * 
+     * - it runs time-aligned to scene ticks
+     * - it runs maximum once per scene-tick
+     * 
+     * @param {Function} cb The function to tickify
+     * @returns {Function)}
+     */
+    tickify(cb) {
```

Comment:
Maybe an `untickify` method as well? Then it can support components that get destroyed during the lifetime of a Viewer.

Reply:
Please let me finish the PR description 😂